cityofaustin/atd-moped
# Bump moped graphql engine & Remove heroku workflows

## Associated issues

This PR intends to close [10220](https://github.com/cityofaustin/atd-data-tech/issues/10220) & [10128](https://github.com/cityofaustin/atd-data-tech/issues/10128). 

## Considerations

Acceptance of this PR indicates an acceptance of bumping the production moped graphql-engine version to 2.11.2 during the next release party, this Thursday.

## Testing
* Spin up your local moped DB stack and see how it works
* Observe that there is no longer a github action for deployment of test to Heroku

---

#### Ship list
